### ISSUE TITLE
pip install konduit fails on Ubuntu 18.04 because jnius is missing

### ISSUE THREAD

**Issue by kmotohas** [KonduitAI/konduit-serving#8]
It seems that the `setup.py` needs to be modified.
looks pydatavec requires pydl4j, and pydl4j requires pyjnius?

```
$ pip install konduit
Collecting konduit
  Downloading https://files.pythonhosted.org/packages/cf/30/2f77ba0ff0b76ddc2796a4c55c1fdd42f7c14d94d6aee44250a67a8a8982/konduit-0.1-py3-none-any.whl
Collecting requests>=2.22.0 (from konduit)
  Using cached https://files.pythonhosted.org/packages/51/bd/23c926cd341ea6b7dd0b2a00aba99ae0f828be89d72b2190f27c11d4b7fb/requests-2.22.0-py2.py3-none-any.whl
Collecting pyarrow==0.13.0 (from konduit)
  Downloading https://files.pythonhosted.org/packages/3f/6c/91a3d949fe0763e60ac181b7b79e74e848e33e402e5e8274cad455519d76/pyarrow-0.13.0-cp37-cp37m-manylinux1_x86_64.whl (48.5MB)
     |████████████████████████████████| 48.5MB 2.2MB/s 
Collecting numpy<=1.16.4 (from konduit)
  Downloading https://files.pythonhosted.org/packages/fc/d1/45be1144b03b6b1e24f9a924f23f66b4ad030d834ad31fb9e5581bd328af/numpy-1.16.4-cp37-cp37m-manylinux1_x86_64.whl (17.3MB)
     |████████████████████████████████| 17.3MB 1.9MB/s 
Collecting pandas<=0.24.2 (from konduit)
  Downloading https://files.pythonhosted.org/packages/22/e6/2d47835f91eb010036be207581fa113fb4e3822ec1b4bafb0d3d105fede6/pandas-0.24.2-cp37-cp37m-manylinux1_x86_64.whl (10.1MB)
     |████████████████████████████████| 10.1MB 2.5MB/s 
Collecting requests-toolbelt>=0.9.1 (from konduit)
  Downloading https://files.pythonhosted.org/packages/60/ef/7681134338fc097acef8d9b2f8abe0458e4d87559c689a8c306d0957ece5/requests_toolbelt-0.9.1-py2.py3-none-any.whl (54kB)
     |████████████████████████████████| 61kB 2.6MB/s 
Requirement already satisfied: certifi>=2017.4.17 in /home/kmotohas/miniconda3/envs/konduit/lib/python3.7/site-packages (from requests>=2.22.0->konduit) (2019.9.11)
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 (from requests>=2.22.0->konduit)
  Downloading https://files.pythonhosted.org/packages/e0/da/55f51ea951e1b7c63a579c09dd7db825bb730ec1fe9c0180fc77bfb31448/urllib3-1.25.6-py2.py3-none-any.whl (125kB)
     |████████████████████████████████| 133kB 2.1MB/s 
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.22.0->konduit)
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting idna<2.9,>=2.5 (from requests>=2.22.0->konduit)
  Using cached https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl
Collecting six>=1.0.0 (from pyarrow==0.13.0->konduit)
  Using cached https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Collecting python-dateutil>=2.5.0 (from pandas<=0.24.2->konduit)
  Using cached https://files.pythonhosted.org/packages/41/17/c62faccbfbd163c7f57f3844689e3a78bae1f403648a6afb1d0866d87fbb/python_dateutil-2.8.0-py2.py3-none-any.whl
Collecting pytz>=2011k (from pandas<=0.24.2->konduit)
  Downloading https://files.pythonhosted.org/packages/e7/f9/f0b53f88060247251bf481fa6ea62cd0d25bf1b11a87888e53ce5b7c8ad2/pytz-2019.3-py2.py3-none-any.whl (509kB)
     |████████████████████████████████| 512kB 1.6MB/s 
ERROR: pydl4j 0.1.3 requires jnius, which is not installed.
Installing collected packages: urllib3, chardet, idna, requests, numpy, six, pyarrow, python-dateutil, pytz, pandas, requests-toolbelt, konduit
Successfully installed chardet-3.0.4 idna-2.8 konduit-0.1 numpy-1.16.4 pandas-0.24.2 pyarrow-0.13.0 python-dateutil-2.8.0 pytz-2019.3 requests-2.22.0 requests-toolbelt-0.9.1 six-1.12.0 urllib3-1.25.6
```

**Comment by maxpumperla** [comment]
right, but `pydl4j` should install it for you. I'm on 18.04 as well and a little surprised :D

https://github.com/eclipse/deeplearning4j/blob/master/pydl4j/setup.py#L24

**Comment by ShamsUlAzeem** [comment]
I've noticed that sometimes it doesn't pick up the items mentioned in the `install_requires` label

**Comment by maxpumperla** [comment]
ok, so we did have dependencies in the `tests` profile, which you can install with

```python
pip install -e ".[tests]"
```

in ubuntu. but in the meantime I released another version, namely `0.1.1` that fixes that issue
